lib/commands/geolocation.js

```diff
@@ -36,16 +36,30 @@ export async function setGeoLocation(location) {
 }
 
 /**
+ * Set the device geolocation.
+ *
  * @this {import('../driver').AndroidDriver}
- * @param {number} latitude
- * @param {number} longitude
- * @param {number} [altitude]
+ * @param {number} latitude Valid latitude value.
+ * @param {number} longitude Valid longitude value.
+ * @param {number} [altitude] Valid altitude value.
+ * @param {number} [satellites] Number of satellites being tracked (1-12).
+ * @param {number} [speed] Valid speed value.
+ * https://developer.android.com/reference/android/location/Location#setSpeed(float)
+ * @param {number} [bearing] Valid bearing value.
+ * https://developer.android.com/reference/android/location/Location#setBearing(float)
+ * @param {number} [accuracy] Valid accuracy value.
+ * https://developer.android.com/reference/android/location/Location#setAccuracy(float),
+ * https://developer.android.com/reference/android/location/Criteria
  */
-export async function mobileSetGeolocation(latitude, longitude, altitude) {
+export async function mobileSetGeolocation(latitude, longitude, altitude, satellites, speed, bearing, accuracy) {
```

Comment:
it probably makes sense to put line breaks between each arg to make the long list better readable